CHANGELOG.md

```diff
@@ -8,8 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### ✅ Added
 - `ChatViewController.preferredEmojiOrder` to specify order of emojis in reaction view [#337](https://github.com/GetStream/stream-chat-swift/pull/337)
-
-### 🔄 Changed
+- `client.unban(user:User in channel: Channel, _ completion:)` [#344](https://github.com/GetStream/stream-chat-swift/pull/337)
```

Comment:
The links are incorrect, they point to 337. Also, can you add a single entry for all these?
```
- `unban` endpoint: [#344]:
  - `client.unban`
  - `channel.unban`
  ...
```